docs/pages/reference/mesheryctl-commands.md

```diff
@@ -465,49 +465,26 @@ Installation, troubleshooting and debugging of Meshery and its adapters
     <th>Flag</th>
     <th>Function</th>
   </tr>
-  {% assign command7 = site.data.mesheryctlcommands.cmds.mesh %}
+  {% assign command7 = site.data.mesheryctlcommands.cmds.adapter %}
     <tr>
-      <td rowspan=13><a href="{{ site.baseurl }}/reference/mesheryctl/mesh">{{ command7.name }}</a></td>
+      <td rowspan=12><a href="{{ site.baseurl }}/reference/mesheryctl/{{ command7.name }}">{{ command7.name }}</a></td>
       <td></td>
       <td></td>
       <td>{{ command7.description }}</td>
     </tr>
-    {% assign subcommand1 = command7.subcommands.validate %}
+    {% for subcommand_hash in command7.subcommands %}{% assign subcommand = subcommand_hash[1] %}
       <tr>
-        <td rowspan=5><a href="{{ site.baseurl }}/reference/mesheryctl/mesh/validate">{{ subcommand1.name }}</a></td>
+        <td rowspan={{ subcommand.flags.size | plus:1 }} ><a href="{{ site.baseurl }}/reference/mesheryctl/{{ command7.name }}/{{ subcommand.name }}">{{ subcommand.name }}</a></td>
         <td></td>
-        <td>{{ subcommand1.description }}</td>
-      </tr>
-      {% for flag_hash in subcommand1.flags %}{% assign flag = flag_hash[1] %}
-        <tr>
-          <td>{{ flag.name }}</td>
-          <td>{{ flag.description }}</td>
-        </tr>
-      {% endfor %}
-    {% assign subcommand2 = command7.subcommands.remove %}
-      <tr>
-        <td rowspan=2><a href="{{ site.baseurl }}/reference/mesheryctl/mesh/remove">{{ subcommand2.name }}</a></td>
-        <td></td>
-        <td>{{ subcommand2.description }}</td>
-      </tr>
-      {% for flag_hash in subcommand2.flags %}{% assign flag = flag_hash[1] %}
-        <tr>
-          <td>{{ flag.name }}</td>
-          <td>{{ flag.description }}</td>
-        </tr>
-      {% endfor %}
-    {% assign subcommand3 = command7.subcommands.deploy %}
-      <tr>
```

Comment:
Any reason for removing these @lekaf974? Since this is renaming the command from `mesh`=>`adapter` commands would still remain same right? No rush, but just asking

Reply:
> Any reason for removing these @lekaf974? Since this is renaming the command from `mesh`=>`adapter` commands would still remain same right? No rush, but just asking

Just use advantage of the templating language to auto generate, for example if we add a new flag or subcommand, no need to think to add it manually it will be displayed in the docs

Reply:
FINALLY. 😄 Thank you, @lekaf974 💯